packages/core/src/Dropdown/tests/dropdown.test.js

```diff
@@ -530,14 +529,11 @@ describe("callbacks", () => {
     expect(onFocusSpy).toHaveBeenCalledTimes(0);
     expect(onBlurSpy).toHaveBeenCalledTimes(0);
 
-    // close dropdown by clicking outside of it
-    const externalButton = getByRole("button", { name: /to click on/i });
-    userEvent.click(externalButton);
+    const externalButton = getByRole("button", { name: /To click on/i });
+    fireEvent.click(externalButton);
```

Comment:
@nantunes somehow i needed to change the event trigger. 
The test was failing with the userEvent,